app/src/main/java/com/ex/service/SomeService.java

```diff
@@ -0,0 +1,12 @@
+package com.ex.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class SomeService {
+	
+	public int add(int x, int y){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AIndentationCheck)